### PR TITLE
Revise description for Concatenate Arrays with .concat Challenge

### LINKED ISSUE #10511: Concatenate with concat, more than one array
#### Challenge Name

https://www.freecodecamp.com/challenges/concatenate-arrays-with-concat
#### Issue Description

The description says that the method can be used to concatenate two arrays. I bet most of us did wonder then how can you concatenate more than two.
I think it is worth mentioning that it can be used to concatenate as many arrays as you want by adding parameters to the method. In fact, the example would gain value by including more arrays. The method is so simple that a more complex example would be an improvement.
#### Browser Information

- Browser Name, Version: 
- Operating System: 
- Mobile, Desktop, or Tablet: 
#### Your Code

``` js



```

### PR DESCRIPTION
**Pull request by johnnyqbui**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #10511

#### Description

Revised description for Concatenate Arrays with .concat challenge for a more accurate description of .concat.

-----
Fixed checklist by @abhisekp - 29th Nov. 2016 - 8:33 AM IST

### CODE REVIEW

**Review by t3h2mas — COMMENTED** [review]
LGTM. IMO array(s) is better left as arrays, since two is the minimum for concatination

**Comment by abhisekp** [comment]
@johnnyqbui How does the challenge description look like? Any screenshot?

**Comment by johnnyqbui** [comment]
@t3h2mas ok, i've changed it to arrays
@abhisekp 
capture

What do you guys think?

File: seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json [hunk 1]
```diff
@@ -563,10 +563,13 @@
...
       "description": [
-        "<code>concat</code> can be used to merge the contents of two arrays into one.",
+        "<code>concat</code> can be used to merge the contents of two or more arrays into one.",
         "<code>concat</code> takes an array as an argument and returns a new array with the elements of this array concatenated onto the end.",
-        "Here is an example of <code>concat</code> being used to concatenate <code>otherArray</code> onto the end of <code>oldArray</code>:",
-        "<code>newArray = oldArray.concat(otherArray);</code>",
+        "Here are examples of <code>concat</code> being used to concatenate arrays:",
+        "Example 1: <code>otherArray</code> onto the end of <code>oldArray</code>:",
+        "<blockquote>newArray = oldArray.concat(otherArray);</blockquote>",
+        "Example 2: <code>newArray</code> with all values of <code>arr1, arr2,</code> and <code>arr3</code>:",
```
Comment (abhisekp) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]:
```diff
- ...with all values of...
+...with all elements of... 
```

**Review by abhisekp — CHANGES_REQUESTED** [review]

File: seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json [hunk 2]
```diff
@@ -563,10 +563,13 @@
...
       "description": [
-        "<code>concat</code> can be used to merge the contents of two arrays into one.",
+        "<code>concat</code> can be used to merge the contents of two or more arrays into one.",
         "<code>concat</code> takes an array as an argument and returns a new array with the elements of this array concatenated onto the end.",
-        "Here is an example of <code>concat</code> being used to concatenate <code>otherArray</code> onto the end of <code>oldArray</code>:",
-        "<code>newArray = oldArray.concat(otherArray);</code>",
+        "Here are examples of <code>concat</code> being used to concatenate arrays:",
+        "Example 1: <code>otherArray</code> onto the end of <code>oldArray</code>:",
+        "<blockquote>newArray = oldArray.concat(otherArray);</blockquote>",
+        "Example 2: <code>newArray</code> with all elements of <code>arr1, arr2,</code> and <code>arr3</code>:",
```
Comment (abhisekp) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]:
```diff
- "Example 2: <code>newArray</code> with all elements of <code>arr1, arr2,</code> and <code>arr3</code>:",
+ "Example 2: <code>newArray</code> with all the elements of <code>arr1</code>, <code>arr2</code>, and <code>arr3</code>:",

**Review by abhisekp — APPROVED** [review]
<h1 align="center"> :sparkles:  :smile_cat:  :+1: LGTM :smiley: :+1:   :sparkles:  </h1>

**Comment by johnnyqbui** [comment]
@abhisekp Yay! my first approved PR, thanks!

**Comment by abhisekp** [comment]
Thanks for this one. :tada: :smiley: 

Let it keep coming. Love your work :heartpulse:

You might want to check out [**Writing Good Git Commit Messages**](https://forum.freecodecamp.com/t/writing-good-git-commit-messages/13210)

Enjoy! :smiley:

**Pull request merged**